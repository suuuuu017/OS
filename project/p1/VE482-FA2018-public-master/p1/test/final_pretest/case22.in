WRITELN cd non-existing
WRITELN exit
