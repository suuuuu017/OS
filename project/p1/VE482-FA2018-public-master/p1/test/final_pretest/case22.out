mumsh $ non-existing: No such file or directory
mumsh $ exit
